### PR TITLE
Solution_v2_

### PR DESCRIPTION
**Pull request by ArtMot99**

### CODE REVIEW

**Comment by ArtMot99** [comment]
What's wrong?! Why is he cursing?!

File: app/main.py [hunk 1]
```diff
@@ -1,3 +1,50 @@
...
+        return Distance(self.km + other_distance)
+
+    def __iadd__(self, other: (int, float)) -> object:
+        other_distance = other if isinstance(other, (int, float)) else other.km
+        self.km += other_distance
+
+        return self
+
+    def __mul__(self, other: (int, float)) -> object:
+        other_distance = other if isinstance(other, (int, float)) else other.km
```
Comment (IvanRamyk) [inline app/main.py:22]: `other` instance couldn't have type `Distance`

File: app/main.py [hunk 2]
```diff
@@ -1,3 +1,50 @@
...
+        return Distance(self.km * other_distance)
+
+    def __truediv__(self, other: (int, float)) -> object:
+        other_distance = other if isinstance(other, (int, float)) else other.km
```
Comment (IvanRamyk) [inline app/main.py:26]: same here

**Review by IvanRamyk — CHANGES_REQUESTED** [review]

**Review by IvanRamyk — APPROVED** [review]
Well done!

**Pull request closed without merging**